WhichSpace/AppDelegate.swift

```diff
@@ -9,13 +9,14 @@
 import Cocoa
 
 @NSApplicationMain
-class AppDelegate: NSObject, NSApplicationDelegate {
+class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
 
     @IBOutlet weak var window: NSWindow!
     @IBOutlet weak var statusMenu: NSMenu!
 
     var icons = [NSImage]()
-    let statusBarItem = NSStatusBar.systemStatusBar().statusItemWithLength(-1)
+    let statusBarItem = NSStatusBar.systemStatusBar().statusItemWithLength(27)
```

Comment:
I'm very new to Cocoa, so excuse my ignorance, but why 27 rather than `NSVariableStatusItemLength` (i.e. `-1`)?